lib/ramble/docs/tutorials/configuring_a_scaling_study.rst

```diff
@@ -0,0 +1,245 @@
+study for `WRF <https://www.mmm.ucar.edu/models/wrf>`_, a free and open-source
+application for atmospheric research and operational forecasting applications.
+
+This tutorial builds off of concepts introduced in previous tutorials. Please
+make sure you review those before starting with this tutorial's content.
+
+Create a Workspace
+------------------
+
+To begin with, you need a workspace to configure the scaling study. This can be
+created with the following command:
+
+.. code-block:: console
+
+    $ ramble workspace create scaling_wrf
+
+
+Activate the Workspace
+----------------------
+
+Several of Ramble's commands require an activated workspace to function
+properly. Activate the newly created workspace using the following command:
+(NOTE: you only need to run this if you do not currently have the workspace
+active).
+
+.. code-block:: console
+
+    $ ramble workspace activate scaling_wrf
+
+Decide on a Workload
+--------------------
+
+Before you can setup this workspace, you'll need to configure the experiments
+you want to execute. To begin with, select a workload from the output of:
+
+.. code-block:: console
+
+    $ ramble info wrfv4
+
+For the purposes of this tutorial, the ``CONUS_12km`` workload is recommended
+because it is less computationally expensive than the ``CONUS_2p5km`` workload.
+
+Configure Experiment Definitions
+--------------------------------
+
+Now that you have selected a workload to use, edit the workspace configuration
+file. The workspace's root location can be seen under the ``Location`` output of:
+
+.. code-block:: console
+
+    $ ramble workspace info
+
+Additionally, the files can be edited directly with:
+
+.. code-block:: console
+
+    $ ramble workspace edit
+
+Within this file, configure the ``applications`` dictionary to describe the
+experiments you want to execute. The contents might look like the following:
+
+.. code-block:: YAML
+
+    ramble:
+      applications:
+        wrfv4:
+          workloads:
+            CONUS_12km:
+              experiments:
+
+
+The next step in configuring the experiment definitions is to decide on an
+experiment name template. For the purposes of this tutorial, we'll assume we
+only want to change the ``n_nodes`` variable definition in our scaling study,
+and as a result the experiment name template will only include this template
+parameter. However, you are free to add additional parameters based on the
+experiments you would like to perform. We will also assume ``n_nodes`` will
+take the values of ``1``, ``2``, and ``4``, however you should edit this for
+the system you are attempting to run these experiments on. The contents of the
+configuration file might look like the following now:
+
+.. code-block:: YAML
+
+    ramble:
+      applications:
+        wrfv4:
+          workloads:
+            CONUS_12km:
+              experiments:
+                scaling_{n_nodes}:
+                  variables:
+                    n_nodes: [1, 2, 4]
```

Comment:
I think in the context of the snippet below its not so bad, but do we want to also add ppn or n_ranks here? I know it complicates the example but this snippet also goes against are best (required) practices of specifying 2 of the 3

Reply:
I can't decide. I was going to add `n_ranks` here, but that just adds the math and then is required to have `processes_per_node` defined as well. And I don't really want it to be defined in the experiment section.

Maybe it'd be good to inject the error that you would see if you tried to use this experiment directly, as a point of "Hey, you need to define these things now."